Made local

LouisRitchie opened the pull request:
#11 

This includes the layout and the SVG of the orange right-facing arrow. Background image is required!

LouisRitchie committed:
first pass at made-local layout

LouisRitchie committed:
add arrow-right.svg; finish up

adamish717 commented:
@LouisRitchie , can I request a placeholder image of some kind? There are several ways to drop images in, and it's easier for me to replace images that exist rather than hoping I know how you want it inserted and styled.

I'll put the new images in Dropbox today.

LouisRitchie committed:
move icon to proper directory

LouisRitchie committed:
add a bg image

LouisRitchie committed:
use existing placeholder image; gradient makes txt readable.

LouisRitchie commented:
I have added an existing image as a placeholder. It is the wrong hue, so the text isn't readable with the image underlaying it. I've fixed this by adding a temporary gradient to overlay the image where the text lies. This has a side-effect of making the icon in this section look funny when it is hovered; this will be rectified when we have a bg image for this section.

LouisRitchie commented:
@gregdaynes I think this one is ready for your review.

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+.made-local-section {
+  @include font-body;
```
is not necessary, will inherit font from the body

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+.made-local-section {
+  @include font-body;
+  font-weight: 100;
```
doesn't have any affect, work sans lightest weight is 300.
You can see all the fonts + weights in `fonts.yml`

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+.made-local-section {
+  @include font-body;
+  font-weight: 100;
+  color: $color-text--dark;
```
not necessary

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+
+  .section__lead {
+    width: 570px;
+    line-height: 30px;
```
please use unitless values for line-height. That way when the font size changes the letters won't overlap / do odd things.
font size for this element was 20px
To convert: 30px / 20px = 1.5

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+  .section__lead {
+    width: 570px;
+    line-height: 30px;
+    margin: 20px auto;
```
on line 10 if you make `.section__header` with `570px` you can remove the width and margin properties here.

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+    width: 570px;
+    line-height: 30px;
+    margin: 20px auto;
+    font-weight: 400;
```
unnecessary

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+.made-local-section {
```
consider naming this `.made-local` and not including section. It lends itself more to component based architecture/bem

following classes can be renamed to `.made-local__header` to contain all their styling to the block

they could also be given a higher level class `.section__header`, `.section__title` for things that match other sections around the site.

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -0,0 +1,38 @@
+  }
+
+  .section__link {
+    @include tiny;
```
not sure if a smaller than body font adds anything here. Not asking for change, but something to consider.

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -33,6 +33,10 @@
+
+      img {
+        opacity: 0.65;
+      }
```
@this does not have the desired highlighting effect.
3 options here
1) use svg/sprite with both a normal and hover color and use background positioning to change them out on hover
2) inline the svg using `{% include icons/right-arrow.svg %}` and change the color with css
```
&:hover {
  fill: $color-text--orange-hover
}
```
3) image tag (which you've done here) and change color with css - I don't think the above css works for it, so you'd have to do some research (I remember something about this still not working well, also requiring base64)

gregdaynes replied:
Further commit changed to the 2nd option - so you can use `fill` to specify the color.

gregdaynes commented on src/assets/css/_sass/components/_home-products.scss:
```diff
@@ -24,6 +24,10 @@
       color: $color-text--orange-hover;
+
+      img {
+        opacity: 0.7;
```
same as home-made-local.scss comment

gregdaynes commented on src/assets/css/_sass/components/_home-made-local.scss:
```diff
@@ -4,7 +4,7 @@
   padding-top: 120px;
   height: 1200px;
-  background-image: url('/assets/images/_MG_0777.jpg');
+  background-image: linear-gradient(rgba(255,255,255,0.7), transparent, transparent), url("/assets/img/wood-3241848_1920.jpg");
```
Sucks that the image is not predominantly white like the mocks. It might be nice to change the text to white here, and the gradient to a dark color.

adamish717 replied:
This is a placeholder image, so we're fine for now. I'll figure out a better image

adamish717 committed:
Fixup Made Local image

LouisRitchie committed:
remove redundant/bad css lines. update arrow-right.svg

LouisRitchie committed:
fix up the text with a drop shadow

LouisRitchie committed:
update arrow-right.svg to have a shadw

LouisRitchie committed:
finish up with _home-made-local.scss

LouisRitchie committed:
a margin

LouisRitchie committed:
clean up some dead CSS

Commit:
Fixup arrow svg
